fmskill/comparison.py

```diff
@@ -843,9 +843,11 @@ def scatter(
             the size of each bin in the 2d histogram, by default None
         nbins : int, optional
             number of bins (if binsize is not given), by default 20
-        show_points : bool, optional
+        show_points : (bool, int, str), optional
             Should the scatter points be displayed?
-            None means: only show points if fewer than threshold, by default None
+            None means: only show points if fewer than threshold, by default None.
```

Comment:
I think we should change the default to showing 10000 samples... do you agree? 
(and I guess that would work also if number of points is less?)